src/container/internal.tsx

```diff
@@ -143,6 +143,7 @@ export default function InternalContainer({
                   [styles['with-toolbar']]: hasToolbar,
                   [styles['with-hidden-content']]: !children || __hiddenContent,
                   [styles['header-with-media']]: hasMedia,
+                  [styles['header-full-page']]: __fullPage && isRefresh,
```

Comment:
This line is the only change compared to the previous PR. Since the full-page variant only exists in VR, we only apply this class here when VR is enabled.